coremltools/models/neural_network/optimization_utils.py

```diff
@@ -124,7 +124,17 @@ def _conv_bn_fusion(conv_idx, bn_idx, layers):
         b = _np.zeros(conv.outputChannels)
 
     w = w.reshape(conv.outputChannels, int(len(w) / conv.outputChannels))
+    if gamma.shape[0] == 0 or variance.shape[0] == 0 or w.shape[0] == 0:
+     print("Skipping fusion due to empty tensor shapes.")
```

Comment:
Thank you for contributing! 
It would be better to use logger instead of print. Something like
```
from coremltools import _logger as logger

...

logger.info("Skipping fusion due to empty tensor shapes.")
```